sentry_sdk/integrations/__init__.py

```diff
@@ -54,6 +54,7 @@ def iter_default_integrations(with_auto_enabling_integrations):
 _AUTO_ENABLING_INTEGRATIONS = (
     "sentry_sdk.integrations.django.DjangoIntegration",
     "sentry_sdk.integrations.flask.FlaskIntegration",
+    "sentry_sdk.integrations.quart.QuartIntegration",
```

Comment:
Not sure we want Quart to be auto enabling quite yet, we can turn it on later based on user feedback. For now let's leave it off?

Reply:
Should I remove all the tests for the auto integration as well?